src/main/java/org/mockito/internal/progress/MockingProgressImpl.java

```diff
@@ -143,4 +146,18 @@ public void setVerificationStrategy(VerificationStrategy strategy) {
     public VerificationMode maybeVerifyLazily(VerificationMode mode) {
         return this.verificationStrategy.maybeVerifyLazily(mode);
     }
-}
+
+    public void setStubbingListener(StubbingListener stubbingListener) {
+        this.stubbingListener = stubbingListener;
+    }
+
+    public StubbingListener getStubbingListener() {
+        if (this.stubbingListener == null) {
+            return new StubbingListener() {
```

Comment:
I would rather set this in the constructor of this class, to prevent needless if checks if there are no listeners (which is the majority of the cases)